projects/observability/src/pages/explorer/explorer.component.ts

```diff
@@ -225,9 +210,31 @@ export class ExplorerComponent {
     this.preferenceService.set(ExplorerComponent.RESULTS_EXPANDED_PREFERENCE, expanded);
   }
 
-  private updateUrlWithVisualizationData(request: ExploreRequestState): void {
+  protected buildContextItems(): Observable<ExplorerContextToggleItem[]> {
```

Comment:
overriding `buildContextItems` in the child class to evaluate a `ff` before populating contexts